devicetypes/smartthings/zwave-door-window-sensor.src/zwave-door-window-sensor.groovy

```diff
@@ -93,6 +97,13 @@ def updated() {
 	response(cmds)
 }
 
+/**
+ * PING is used by Device-Watch in attempt to reach the Device
+ * */
+def ping() {
```

Comment:
@skt123 Just got word from @mckeed that this is a Sleepy device. 
Do not implement `ping` here.